Makefile.am

```diff
@@ -144,7 +144,9 @@ EXTRA_DIST+= INSTALL
 EXTRA_DIST+= IPP
 EXTRA_DIST+= LPCExpresso.cproject
 EXTRA_DIST+= LPCExpresso.project
-EXTRA_DIST+= resource.h wolfssl.rc
+EXTRA_DIST+= resource.h
```

Comment:
this doesn't look relevant to the cmake changes. Why is it here? If its needed maybe a separate PR.